File: OPENAPI-REST-API/swagger-client/dart/lib/model/quickserver_ip_info.dart
part of swagger.api;

class QuickserverIpInfo {
  /* Table title */
  String title = null;

  List<QuickserverIpTableRow> rows = [];

  QuickserverIpInfo();

  @override
  String toString() {
    return 'QuickserverIpInfo[title=$title, rows=$rows, ]';
  }

  QuickserverIpInfo.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    title = json['title'];
    rows = QuickserverIpTableRow.listFromJson(json['rows']);
  }

  Map<String, dynamic> toJson() {
    return {
      'title': title,
      'rows': rows
     };
  }

  static List<QuickserverIpInfo> listFromJson(List<dynamic> json) {
    return json == null ? new List<QuickserverIpInfo>() : json.map((value) => new QuickserverIpInfo.fromJson(value)).toList();
  }

  static Map<String, QuickserverIpInfo> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, QuickserverIpInfo>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new QuickserverIpInfo.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/QuickserverIpInfo.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// IP address information table for a QuickServer service.
  /// </summary>
  [DataContract]
  public class QuickserverIpInfo {
    /// <summary>
    /// Table title
    /// </summary>
    /// <value>Table title</value>
    [DataMember(Name="title", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "title")]
    public string Title { get; set; }

    /// <summary>
    /// Gets or Sets Rows
    /// </summary>
    [DataMember(Name="rows", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "rows")]
    public List<QuickserverIpTableRow> Rows { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class QuickserverIpInfo {\n");
      sb.Append("  Title: ").Append(Title).Append("\n");
      sb.Append("  Rows: ").Append(Rows).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
